Source/Streamstone.Tests/Scenarios/Tracking_entity_changes.cs

```diff
@@ -577,15 +586,21 @@ yield return
             }
         }
 
-        void InsertTestEntity(TestEntity entity)
+        void InsertTestEntity(ITableEntity entity)
```

Comment:
Otherwise, something like `DynamicTableEntity` need to be used for merges. In this way we can have single instance of entity but variable number of properties, so we can show that merge is correctly performed in memory and at storage level. What do you think? @Scooletz